pkg/wwan/mm-init.sh

```diff
@@ -34,12 +34,12 @@ mkdir -p /var/run/dbus
 dbus-daemon --system
 echo "D-Bus daemon started"
 
-echo "Starting Udev daemon"
-udevd --debug --daemon 2>/dev/null
 # Apply installed ModemManager udev rules.
+echo "Setting up udev rules"
```

Comment:
First the rules are copied from `/hostfs/lib/udev` to `/lib/udev` and then they are copied again from `/lib/udev` to `/run/udev`? Isn't it possible to tell udev to directly use the rules from `/hostfs/lib/udev`?
Or are we changing the rules during the runtime of EVE?

Reply:
This is the wwan container, the rules are inside /lib/udev within the container, that's not those inside the /hostfs/lib/udev... here we just copy from wwan container to /run/udev (shared)....

At the initialization of udev service container, we just copy from /hostfs/lib/udev to udev's container /lib/udev.... they are copied only once....

udev only supports the following directories for rules: /etc/udev/rules.d, /run/udev/rules.d and /lib/udev/rules.d (or /usr/lib/udev/rules.d, if enabled during building)....